src/applications/find-forms/helpers/index.js

```diff
@@ -1,38 +1,75 @@
+// dependencies
+import moment from 'moment';
+
+// relative imports
 import { deriveLatestIssue } from '../components/SearchResult';
 import { SORT_OPTIONS } from '../constants';
-
 /**
  * This function sorts the results of Find Forms search.
  * @param {string} sortByPropertyName Is the state property in the SearchResults Controller
  * @param {object} indexA Is a Search Result returned.
  * @param {object} indexB Is a Search Result returned.
  */
-
 export const sortTheResults = (sortByPropertyName, indexA, indexB) => {
   // -n (negative number) sorts indexA to the front of the array.
   // n (positive number) sorts indexA to the back of the array.
-  // 0 keeps both indexA and indexB right where they are.
-  const [LAST_UPDATED_NEWEST_OPTION, LAST_UPDATED_OLDEST_OPTION] = SORT_OPTIONS;
-
+  // stayPut keeps both indexA and indexB right where they are.
+  const [
+    ALPHA_ASCENDING,
+    ALPHA_DESCENDING,
+    LAST_UPDATED_NEWEST_OPTION,
+    LAST_UPDATED_OLDEST_OPTION,
+  ] = SORT_OPTIONS;
+  const sortIndexToFront = -1;
+  const sortIndexToBack = 1;
+  const indexRemainsInPlace = 0;
   const latestTimeStampIndexA = deriveLatestIssue(
     indexA.attributes.firstIssuedOn,
     indexA.attributes.lastRevisionOn,
   );
-
   const latestTimeStampIndexB = deriveLatestIssue(
     indexB.attributes.firstIssuedOn,
     indexB.attributes.lastRevisionOn,
   );
 
+  // SORT BY DATE
+  // ASCENDING
+  if (sortByPropertyName === LAST_UPDATED_OLDEST_OPTION) {
+    return (
+      moment(latestTimeStampIndexA, 'MM-DD-YYYY').toDate() -
+      moment(latestTimeStampIndexB, 'MM-DD-YYYY').toDate()
+    );
+  }
+
   // DESCENDING
   if (sortByPropertyName === LAST_UPDATED_NEWEST_OPTION) {
-    return new Date(latestTimeStampIndexB) - new Date(latestTimeStampIndexA);
+    return (
+      moment(latestTimeStampIndexB, 'MM-DD-YYYY').toDate() -
+      moment(latestTimeStampIndexA, 'MM-DD-YYYY').toDate()
+    );
```

Comment:
It looks like `LAST_UPDATED_NEWEST_OPTION` is just the inverse of `LAST_UPDATED_OLDEST_OPTION`, so you could kinda do the same thing as the ALPA sort

Reply:
Oh good point! That will cut down some lines as well.